packages/ui-components/src/components/StyleProvider/StyleProvider.component.tsx

```diff
@@ -149,7 +149,7 @@ export interface StyleProviderProps {
   /** What element to render as a wrapper, respectively where to render the StyleProvider.  */
   stylesWrapper?: StyleProviderStylesWrapper
   /** The name of the theme to render. */
-  theme?: string
+  theme?: string | null
```

Comment:
See comment above. Change to
```suggestion
  theme?: "theme-dark" | "theme-light"
```
Though ideally, since we use this in two places we define the type once and use it in both places.

Reply:
Indeed, I'd suggest a **custom type**, as [above](https://github.com/cloudoperators/juno/pull/547#issuecomment-2449722013) .

Reply:
As discussed f2f - I changed the type